wagtail/wagtailadmin/static_src/wagtailadmin/scss/components/_main-nav.scss

```diff
@@ -121,10 +121,19 @@ $footer-submenu: $submenu-color;
     .account {
         display: none;
     }
+
+    .submenu-trigger:after {
+        width: auto;
+        transition: transform .3s ease;
```

Comment:
Could you use the `transition` mixin instead of the property directly? eg. `@include transition(transform .3s ease);`. The mixin isn't documented unfortunately, but I would like to keep the pattern going.